src/Spider/AbstractSpider.php

```diff
@@ -331,10 +331,8 @@ public function approximateSize(string $size): int
         }
         $size = (float) $m[1];
         switch (strtoupper($m[2])) {
-            case 'GB':
-                $size *= 1024;
             case 'MB':
-                $size *= 1024;
+            case 'GB':
```

Comment:
no, it's incorrect - in case gb we run 3 times, in case mb - 2 times

Reply:
Changes reverted :)